src/søknad/steg/1-omdeg/sivilstatus/begrunnelse/SøknadsBegrunnelse.tsx

```diff
@@ -40,8 +46,80 @@ const Søknadsbegrunnelse: FC<Props> = ({
     datoForSamlivsbrudd,
     datoFlyttetFraHverandre,
     datoEndretSamvær,
+    tidligereSamboerDetaljer,
   } = sivilstatus;
 
+  const [erGyldigIdent, settGyldigIdent] = useState<boolean>(false);
```

Comment:
hvordan blir initiell status her når man går frem og tilbake mellom stegene?

Reply:
Hmm si det, kanskje man skulle lagret en boolean i context for alle de stedene der man bruker identEllerFødselsdatoKomponenten 🤔

Reply:
Testet dette, og så også at når jeg fyller ut fødselsnummer for så å komme tilbake til steget etter å ha vært innom et annet, så har siste sifferet i nummeret forsvunnet fra inputfeltet

Reply:
FIkset i d3cd4f0